src/process-effects.ts

```diff
@@ -0,0 +1,34 @@
+import { TRAM_EFFECT_STORE, TRAM_EFFECT_QUEUE } from './engine-names';
+import { TRAM_TAG_NEW_EFFECTS } from './node-names';
+import { getEffectStore, clearEffectStore, restoreEffectStore } from './effect-store';
+import { TramOneComponent } from './types';
+
+/**
+ * This is a helper function for the dom creation.
+ * This function stores any effects generated when building a tag in resulting node that is generated.
+ *
+ * These are later processed by the mutation-observer, and cleaned up when the node is removed by the mutation-observer.
+ */
+export default (tagFunction: TramOneComponent) => {
```

Comment:
Previously this function verified that the function returned an Element, but now we get that with Typescript!